examples/langchain-typescript-simple/package.json

```diff
@@ -1,8 +1,13 @@
 {
+  "scripts": {
+    "start": "tsx main.ts"
+  },
   "devDependencies": {
-    "typescript": "^5.2.2"
+    "tsx": "^4.6.2",
```

Comment:
More for my understanding - is `tsx` or `ts-node` better?

Reply:
tsx. ts-node is the original approach and has problems with a mix of imports vs requires. As soon as you start using some of the libraries that have incorporated Ollama, you hit this problem and then you go down a rabbit hole of bad advice and mess up you tsconfig, package.json and more searching for the right combo that fixes things. Bun does even better, but is a heavier install.